### PR TITLE
Add flag to disable TLS for live reload of templ components

### PR DESCRIPTION
**Pull request by nickclyde**
This adds an additional config flag `tls` for disabling serving with TLS. This enables the use of Templ's proxy feature for live reload, meaning we can make changes to our templ components, and the browser will refresh the page automatically, making frontend dev much more seamless!

I also added a `.env.sample` file with some sensible defaults.

### CODE REVIEW

**Review by katyasoup — COMMENTED** [review]
This is great! It doesn't seem to be picking up stylesheet changes though, is it possible to add that too? See video below - template changes (like adding or removing a class) work as expected, but I have to manually start/stop for css changes to take effect.

https://github.com/user-attachments/assets/30d50d55-1862-4a28-a306-a02d7b067af0

File: README.md [hunk 1]
```diff
@@ -41,8 +41,14 @@ PHIN VADS written in Go. Load `phinvads.dump` into a PostgreSQL database using `
...
+1. Run the app! If you are only working on backend code, you can just run a simple live reload with air:
 
     ```bash
     air
     ```
+
+1. Air will also work for the frontend, but you will have to refresh your browser every time you make a change. To get automatic browser reloads, run the app this way:
+
+    ```bash
+    templ generate --watch --proxy="http://localhost:4000" --cmd="go run ./cmd/phinvads-go -tls=false"
```
Comment (akintner) [inline README.md:53]: @katyasoup was that behavior in your comment just from running `air` or was it from this  ^ whole command?

**Comment by nickclyde** [comment]
Okay @katyasoup I think I managed to get hot reloading for static asset changes working as well - can you test it out and let me know if it's working for you?

File: README.md (hunk 1 above)
Comment (katyasoup) [reply README.md]: the `templ generate ...` from the readme

**Review by katyasoup — APPROVED** [review]
Working great now! 🎉

**Pull request merged**